clustering/faces/mojarra/src/main/java/org/wildfly/clustering/faces/mojarra/facelets/el/FaceletsELSerializationContextInitializer.java

```diff
@@ -10,14 +10,16 @@
 import org.wildfly.clustering.marshalling.protostream.SerializationContext;
 import org.wildfly.clustering.marshalling.protostream.SerializationContextInitializer;
 
+import com.sun.faces.facelets.el.ELText;
```

Comment:
nitpick only: this should just be along the other imports.

Reply:
Fixed